contextualized/regression/__init__.py

```diff
@@ -9,14 +9,15 @@
 )
 from contextualized.regression.losses import MSE, BCELoss
 from contextualized.regression.regularizers import REGULARIZERS
-from contextualized.regression.lightning_modules import (
-    NaiveContextualizedRegression,
-    ContextualizedRegression,
-    MultitaskContextualizedRegression,
-    TasksplitContextualizedRegression,
-    ContextualizedUnivariateRegression,
-    TasksplitContextualizedUnivariateRegression,
-)
+
+# from contextualized.regression.lightning_modules import (
```

Comment:
Should this be commented?

Reply:
As of now, I think this creates a circular import with lightning modules => datamodules => (calls init to get datasets) => lightning modules

Reply:
Ah, so these are user-facing (to allow things like `from contextualized.regression import *`) and shouldn't be used internally. Make sure your imports are structured as `from contextualized.regression.<module> import <thing>` rather than `from contextualized.regression import <thing>`. The latter leads to cyclic imports. I saw one example of this on line 7 of your datamodules.py

Reply:
Yes, we should definitely apply the same pattern we have in `modules.py` to all of these "module summary" lists/dicts! But rather than making a new `helpers.py` file, they should each live in the file where their components are defined. I've made an issue #154 to track this. Feel free to tackle it in this PR if you feel it's relevant.